test/gtest/stringify.cpp

```diff
@@ -125,3 +123,100 @@ adding unique symbol
 
   EXPECT_EQ(ss.str(), stringifyText);
 }
+
+using HashStringifyTest = PrintTest;
+
+TEST_F(HashStringifyTest, Print) {
```

Comment:
I think this can just be lumped into the same test suite as the previous test, but it does have to have a new name.
```suggestion
TEST_F(StringifyTest, Stringify) {
```

Reply:
done